oclint-scripts/ci

```diff
@@ -17,8 +17,13 @@ arg_parser.add_argument('-setup', '--setup', action="store_true")
 arg_parser.add_argument('-llvm-branch', '--llvm-branch', nargs='?', choices=version.llvm_branches(), default=version.llvm_default_branch())
 arg_parser.add_argument('-release', '--release', action="store_true")
 arg_parser.add_argument('-archive', '--archive', action="store_true")
+arg_parser.add_argument('-j', type=int, default=0)
```

Comment:
Why can't we leave it default? This script is for Continuous Integration (it was once Jenkins, and now Travis CI), and maybe we can leave it to the CI environment?